week-2/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,10 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (num > 0);
+  return true;
+}
```

Comment:
There are a couple of issues here:
1) You're returning `true` if the given number is *greater* than 0. Should it be the opposite?
2) whatever word you choose to name your parameter, you *must* repeat the same word inside the function. So here, if your param is named *number*, you should be checking if `number` is greater than / lower than `0`, not `num` - the function has no idea what `num` means.